README.md

```diff
@@ -5,6 +5,8 @@
 
 Drone plugin for deploying to Cloud Foundry
 
+See [DOCS.md](DOCS.md) for `.drone.yml` usage
```

Comment:
I would suggest adding this to all the other plugins. The difference between _how to use this in development_ and _how to use this in practice_ should both be readily available in the readme. I actually think _how to use this in practice_ is more important and should be the one in `README.md`. Whereas `DOCS` would make more sense for development